magma/lib/magma/flag.rb

```diff
@@ -1,5 +1,18 @@
 class Magma
   class Flag < Sequel::Model
+    # update_or_create is natively implemented in a later version of Sequel - I believe it can also be installed as a plugin
+    def self.update_or_create(project_name, flag_list)
```

Comment:
Should this also include some check that the flag name and value are valid for production (i.e., anything in `module Flags` on l.18)?

Reply:
this is a good question. Lets chat about this briefly in stand-up.